code/mob/living.dm

```diff
@@ -1920,7 +1920,8 @@ var/global/icon/human_static_base_idiocy_bullshit_crap = icon('icons/mob/human.d
 /mob/living/shock(var/atom/origin, var/wattage, var/zone = "chest", var/stun_multiplier = 1, var/ignore_gloves = 0)
 	if (!wattage)
 		return 0
-
+	if (isdead(src) && !src.bioHolder.HasEffect("resist_electric"))
+		src.setStatus("defibbed", 1.5 SECONDS)
```

Comment:
remove `if` and move to right above the shock damage switch

Reply:
i think i did it